### ISSUE TITLE
Demo site editing a PDF with Adobe doesn't seem to work any more

### ISSUE THREAD

**Issue by catmanjan** [ITHit/WebDAVAjax#3]
I recall editing a PDF from the demo site before, but now it doesn't seem to work with Adobe Reader:

![image](https://github.com/ITHit/WebDAVAjax/assets/137350/3e2db561-62f7-4272-b4c4-840f678af869)

I see a GET request, but when I try to save I don't see any LOCKs or UNLOCKs and instead I am prompted to save to Adobe cloud

**Comment by ITHitBuild** [comment]
Could you please clarify how you edit the document with Adobe Reader? Because you typically need Acrobat Pro or Acrobat Standard to edit the document.

I have just double checked PDF editing using Adobe Acrobat Pro on both https://webdavserver.com and https://webdavserver.net and everything worked as expected.

**Comment by catmanjan** [comment]
@ITHitBuild I've attached a PDF which is a form, with Adobe Reader I am able to fill in the form and save changes
[test.pdf](https://github.com/ITHit/WebDAVAjax/files/13667991/test.pdf)

**Comment by dimnorin** [comment]
@catmanjan I've checked editing of your test.pdf from ITHit Webdav Demo server https://webdavserver.net/ with this version of Acrobat Reader
![image](https://github.com/ITHit/WebDAVAjax/assets/6441757/585bb923-5208-40b8-91fa-7992779e3408)
I've been asked to save the edited file to the mounted folder of the Demo server. But after choosing, it locks and unlocks the file and saves changes as it should.

**Comment by ITHitBuild** [comment]
Just to clarify. We have retested PDF forms filling with latest versions of Adobe Acrobat Reader and with Adobe Acrobat Pro. In both cases saving PDF file with form work as expected, we did not find any issues. 

- **In case of Adobe Acrobat Reader**, when you save the PDF form opened from WebDAV server, the 'Save As' dialog is displayed, which requires saving into the new file (or overwriting existing file). This behavor is identical to how Adobe reader works with a PDF form opened local file system. I can guess that this is because Adobe Reader do not want the user to accidentally overwrite the original PDF form file.

- **In case of Adobe Acrobat Pro** the PDF form is saved directly back to the server.
